### PR TITLE
Weapons descriptions fix

### PR DESCRIPTION
**Pull request by Walfanger**
I woke up in the morning and decided that it would be nice to make some corrections in descriptions of weapons. 
So... I've done it. Don't you want to take a look?

### CODE REVIEW

File: code/game/objects/items/weapons/melee/energy.dm [hunk 1]
```diff
@@ -168,7 +168,7 @@
 //Can't be activated or deactivated, so no reason to be a subtype of energy
 /obj/item/weapon/melee/energy/blade
 	name = "energy blade"
-	desc = "A concentrated beam of energy in the shape of a blade. Very stylish... and lethal."
```
Comment (NanakoAC) [inline code/game/objects/items/weapons/melee/energy.dm]: This is definitely worse, it was correct before and you've added incorrect grammar

File: code/game/objects/items/weapons/melee/misc.dm [hunk 2]
```diff
@@ -14,7 +14,7 @@
 
 /obj/item/weapon/melee/toolbox_maul
 	name = "toolmop the maul"
-	desc = "Toolbox tied to mop. A weapon of choice."
```
Comment (NanakoAC) [inline code/game/objects/items/weapons/melee/misc.dm]: Also wrong, "the" shouldn't be used here, this description was fine

File: code/modules/projectiles/ammunition/boxes.dm [hunk 3]
```diff
@@ -1,7 +1,7 @@
 /*This code for boxes with ammo, you cant use them as magazines, but should be able to fill magazines with them.*/
 /obj/item/ammo_magazine/ammobox	//Should not be used bu its own
 	name = "ammunition box"
-	desc = "Gun ammunition puted in shiny new box. You can see caliber details on on the label."
```
Comment (NanakoAC) [inline code/modules/projectiles/ammunition/boxes.dm]: try "stored in a shiny new box"

File: code/modules/projectiles/guns/energy/laser.dm [hunk 4]
```diff
@@ -17,15 +17,15 @@
...
-	desc = "A modified version of the NeoTheology brand laser carbine, this one fires less concentrated energy bolts designed for target practice."
+	desc = "A modified version of \"NeoTheology\" brand laser carbine, this one fires less concentrated energy bolts, designed for target practice."
 	matter = list(MATERIAL_PLASTEEL = 20, MATERIAL_WOOD = 8, MATERIAL_SILVER = 2)
 	projectile_type = /obj/item/projectile/beam/practice
 
 obj/item/weapon/gun/energy/retro
 	name = "Alliance LG \"Cog\""
 	icon_state = "retro"
 	item_state = "retro"
-	desc = "Alliance cheap laser gun. In our deep past - this was main weapon of low-rank police force across all Milky Way, bilions copys of this gun was made. They still can be found almoust everywhere."
```
Comment (NanakoAC) [inline code/modules/projectiles/guns/energy/laser.dm]: Improved, but not quite right. Here's a better version

 "Alliance cheap laser gun. In the distant past - this was the main weapon of low-rank police forces across all the Milky Way, billions of copies of this gun were made. They are ubiquitous."

File: code/modules/projectiles/guns/energy/laser.dm [hunk 5]
```diff
@@ -36,7 +36,7 @@ obj/item/weapon/gun/energy/retro
 	name = "NT LG \"Destiny\""
 	icon_state = "caplaser"
 	item_state = "caplaser"
-	desc = "This weapon is old, yet still robust and reliable. Its marked with old Nanotrasen brand, like a last reminder of what this corporation was, before church take control of everything."
+	desc = "This weapon is old, yet still robust and reliable. It's marked with old Nanotrasen brand, like a last reminder of what this corporation was, before the Church take control of everything."
```
Comment (NanakoAC) [inline code/modules/projectiles/guns/energy/laser.dm]: Improved but not perfect, try this

"This weapon is old, yet still robust and reliable. It's marked with an old Nanotrasen brand, a distant reminder of what this corporation was before the Church took control of everything."

File: code/modules/projectiles/guns/energy/laser.dm [hunk 6]
```diff
@@ -47,7 +47,7 @@ obj/item/weapon/gun/energy/retro
 
 /obj/item/weapon/gun/energy/lasercannon
 	name = "Prototype: laser cannon"
-	desc = "With the laser cannon, the lasing medium is enclosed in a tube lined with uranium-235 and subjected to high neutron flux in a nuclear reactor core. This incredible technology may help YOU achieve high excitation rates with small laser volumes!"
+	desc = "With the laser cannon, the lasing medium is enclosed in a tube lined with uranium-235 and subjected to high neutron flux in a nuclear reactor core. This incredible technology may help YOU achieve the high excitation rates with the small laser volumes!"
```
Comment (NanakoAC) [inline code/modules/projectiles/guns/energy/laser.dm]: This is definitely worse, revert this line completely, was fine before

File: code/modules/projectiles/guns/energy/sniperrifle.dm [hunk 7]
```diff
@@ -1,6 +1,6 @@
 /obj/item/weapon/gun/energy/sniperrifle
 	name = "NT MER \"Valkyrie\""
-	desc = "Valkyrie is an older design of Nanotrasen, Lightfall was based on it. A designated marksman rifle capable of shooting powerful ionized beams, this is a weapon to kill from a distance."
+	desc = "\"Valkyrie\" is an old design of Nanotrasen, \"Lightfall\" was based on it. A designated marksman rifle capable of shooting powerful ionized beams, this is a weapon for killing from a distance."
```
Comment (NanakoAC) [inline code/modules/projectiles/guns/energy/sniperrifle.dm]: older was better than old, the rest of the changes in this one are good

File: code/modules/projectiles/guns/energy/stun.dm [hunk 8]
```diff
@@ -18,7 +18,7 @@
 
 /obj/item/weapon/gun/energy/stunrevolver
 	name = "NT SP \"Zeus\""
-	desc = "Also know as stunrevolver. Older and less precise Nanotrasen solution for non-lethal takedowns. This gun have smaller capacity in exchange for S-cells use."
+	desc = "Also know as stunrevolver. Old and imprecise Nanotrasen solution for non-lethal takedowns. This gun has smaller capacity in exchange for S-cells use."
```
Comment (NanakoAC) [inline code/modules/projectiles/guns/energy/stun.dm]: first two changes were better before.

has is correct though

File: code/modules/projectiles/guns/launcher/syringe_gun.dm [hunk 9]
```diff
@@ -63,7 +63,7 @@
 
 /obj/item/weapon/gun/launcher/syringe
 	name = "syringe gun"
-	desc = "A spring loaded rifle designed to fit syringes, designed to incapacitate unruly patients from a distance."
+	desc = "A spring-loaded rifle designed to fit syringes designed to incapacitate unruly patients from a distance."
```
Comment (NanakoAC) [inline code/modules/projectiles/guns/launcher/syringe_gun.dm]: this whole line is bad, having "designed" twice in succession like that sounds wrong, try rewriting it completely

File: code/modules/projectiles/guns/projectile/automatic.dm [hunk 10]
```diff
@@ -1,6 +1,6 @@
 /obj/item/weapon/gun/projectile/automatic //Hopefully someone will find a way to make these fire in bursts or something. --Superxpdude
 	name = "prototype SMG"
-	desc = "A protoype lightweight, fast firing gun. Uses 9mm rounds."
+	desc = "A prototype of a lightweight, fast firing gun. Uses 9mm rounds."
```
Comment (NanakoAC) [inline code/modules/projectiles/guns/projectile/automatic.dm]: this was better before, revert

File: code/modules/projectiles/guns/projectile/automatic/ak47.dm [hunk 11]
```diff
@@ -1,8 +1,8 @@
 /obj/item/weapon/gun/projectile/automatic/ak47
 	name = "Excelsior 7.62x39 AKMS"
-	desc = "Weapon of oppressed, oppressors and just crazy terrorists.\
-		 If it doesn't work, you can always hit him with it! It is the really old designed assault rifle, chambered in 7,62x39.\
-		 It is known for it easy maintaining and low price. This gun is not used by military anymore, but it found a wide spread within criminals and insurgents."
+	desc = "Weapon of an oppressed, oppressors, and just crazy terrorists.\
```
Comment (NanakoAC) [inline code/modules/projectiles/guns/projectile/automatic/ak47.dm]: ouch what a mess. The original was wrong, your edit is wrong but in different ways. Try this...
"Weapon of the oppressed, oppressors, and extremists of all flavors.\

 This is an ancient automatic rifle, chambered in 7.62x39. If it won't fire, percussive maintenance should get it working again.\

It is known for its easy maintainance, and low price. This gun is not in active military service, but has become ubiquitous among criminals and insurgents."

File: code/modules/projectiles/guns/projectile/automatic/c20r.dm [hunk 12]
```diff
@@ -1,6 +1,6 @@
 /obj/item/weapon/gun/projectile/automatic/c20r
 	name = "C-20r"
-	desc = "The C-20r is a lightweight and rapid firing SMG, for when you REALLY need someone dead. Uses 10mm rounds. Has a 'Scarborough Arms - Per falcis, per pravitas' buttstamp."
+	desc = "The C-20r is a lightweight and rapid-firing SMG, when you REALLY need someone dead. Uses 10mm rounds. Has a 'Scarborough Arms - Per falcis, per pravitas' buttstock stamp."
```
Comment (NanakoAC) [inline code/modules/projectiles/guns/projectile/automatic/c20r.dm]: "for when you really need someone dead" was correct, restore that part.

File: code/modules/projectiles/guns/projectile/automatic/dallas.dm [hunk 13]
```diff
@@ -2,8 +2,8 @@
 
 /obj/item/weapon/gun/projectile/automatic/dallas
 	name = "PAR 10mm x 24 \"Dallas\""
-	desc = "Dallas is a pulse-action air-cooled automatic assault rifle made by unknown manufacturer. This weapon is very rare, but deadly efficient. \
-		It's used by elite mercenaries, assasins or bald marines."
+	desc = "Dallas is a pulse-action air-cooled automatic assault rifle made by an unknown manufacturer. This is a very rare, but a deadly efficient one. \
```
Comment (NanakoAC) [inline code/modules/projectiles/guns/projectile/automatic/dallas.dm]: this description was fine before, revert

File: code/modules/projectiles/guns/projectile/automatic/sts35.dm [hunk 14]
```diff
@@ -1,6 +1,6 @@
 /obj/item/weapon/gun/projectile/automatic/sts35
 	name = "STS-35"
```
Comment (NanakoAC) [inline code/modules/projectiles/guns/projectile/automatic/sts35.dm]: remove "the" preceding frontier worlds

File: code/modules/projectiles/guns/projectile/automatic/z8.dm [hunk 15]
```diff
@@ -1,6 +1,6 @@
 /obj/item/weapon/gun/projectile/automatic/z8
 	name = "FS AR 5.56x45mm \"Z8 Bulldog\""
-	desc = "The Z8 Bulldog is an older model bullpup carbine, made by Frozen Star. Uses armor piercing 5.56mm rounds. Makes you feel like a space marine when you hold it."
+	desc = "The Z8 Bulldog is an old model bullpup carbine, made by \"Frozen Star\". It uses armor piercing 5.56mm rounds. Makes you feel like a space marine when you hold it."
```
Comment (NanakoAC) [inline code/modules/projectiles/guns/projectile/automatic/z8.dm]: this was fine before, revert

File: code/modules/projectiles/guns/projectile/pistol/deagle.dm [hunk 16]
```diff
@@ -1,6 +1,6 @@
 /obj/item/weapon/gun/projectile/deagle
 	name = "FS HG .50 \"Avasarala\""
-	desc = "Obvious remake of old Earth \"Desert Eagle\". Robust and straight, this is a gun for a leader, not just an officer."
+	desc = "An obvious replica of an old Earth \"Desert Eagle\". Robust and straight, this is a gun of a leader, not just an officer."
```
Comment (NanakoAC) [inline code/modules/projectiles/guns/projectile/pistol/deagle.dm]: for was more correct than of, revert the second sentance

File: code/modules/projectiles/guns/projectile/pistol/giskard.dm [hunk 17]
```diff
@@ -1,6 +1,6 @@
 /obj/item/weapon/gun/projectile/giskard
 	name = "FS HG .32 \"Giskard\""
-	desc = "That's the 'Frozen Star' popular traumatic pistol. Can even fit into the pocket! Uses .32 rounds."
+	desc = "That's the \"Frozen Star\" popular traumatic pistol. You can even fit into your pocket! Uses .32 rounds."
```
Comment (NanakoAC) [inline code/modules/projectiles/guns/projectile/pistol/giskard.dm]: this was better before, revert

File: code/modules/projectiles/guns/projectile/pistol/gyropistol.dm [hunk 18]
```diff
@@ -1,6 +1,6 @@
 /obj/item/weapon/gun/projectile/gyropistol
 	name = "NT GP \"Zeus\""
-	desc = "A bulky pistol designed to fire self propelled rounds"
+	desc = "A bulky pistol, designed to fire self-propelled rounds"
```
Comment (NanakoAC) [inline code/modules/projectiles/guns/projectile/pistol/gyropistol.dm]: unnecessary comma, revert

File: code/modules/projectiles/guns/projectile/pistol/olivaw.dm [hunk 19]
```diff
@@ -1,6 +1,6 @@
 /obj/item/weapon/gun/projectile/olivaw
 	name = "FS HG .32 \"Olivaw\""
-	desc = "That's the 'Frozen Star' popular traumatic pistol. This one seems to have a two-round burst-fire mode. Uses .32 rounds."
+	desc = "That's a popular traumatic pistol of \"Frozen Star\". This one seems to have a two-round burst-fire mode. Uses .32 rounds."
```
Comment (NanakoAC) [inline code/modules/projectiles/guns/projectile/pistol/olivaw.dm]: this whole description is a mess, rewrite it entirely.
traumatic pistol makes no sense

File: code/modules/projectiles/guns/projectile/revolver/deckard.dm [hunk 20]
```diff
@@ -1,6 +1,6 @@
 /obj/item/weapon/gun/projectile/revolver/deckard
 	name = "FS REV .44 \"Deckard\""
-	desc = "A rare, custom-built revolver. Use when there is no time for Voight-Kampff test."
+	desc = "A rare, custom-built revolver. Use it when there is no time for Voight-Kampff test."
```
Comment (NanakoAC) [inline code/modules/projectiles/guns/projectile/revolver/deckard.dm]: this was better before, revert

File: code/modules/projectiles/guns/projectile/revolver/revolver.dm [hunk 21]
```diff
@@ -1,6 +1,6 @@
 /obj/item/weapon/gun/projectile/revolver
 	name = "FS REV .357 \"Miller\""
-	desc = "The Frozen Star's Miller is a choice revolver for when you absolutely, positively need to put a hole in the other guy. Uses old school .357 ammo."
+	desc = "The \"Miller\" of the \"Frozen Star\" is a revolver of choice when you absolutely, positively need to make a hole in someone. Uses old-school .357 ammo."
```
Comment (NanakoAC) [inline code/modules/projectiles/guns/projectile/revolver/revolver.dm]: change "\"Miller\" of the \"Frozen Star\""
to  \"Frozen Star\" \"Miller\

File: code/modules/projectiles/guns/projectile/shotgun/bull.dm [hunk 22]
```diff
@@ -1,6 +1,6 @@
 /obj/item/weapon/gun/projectile/shotgun/bull
 	name = "FS SG \"Bull\""
-	desc = "A Frozen Star double-barreled pump action shotgun. Marvel of engineering, this gun  often use by police SWAT teams."
+	desc = "A double-barreled pump action shotgun of \"Frosen Star\". Marvel of engineering, this gun is often use by SWAT teams."
```
Comment (NanakoAC) [inline code/modules/projectiles/guns/projectile/shotgun/bull.dm]: very wrong, you missspelled frozen star, revert it completely and rethink

Often use should be Often Used
DO police or swat teams even exist anymore? If not, make it a historical reference

File: code/modules/projectiles/guns/projectile/shotgun/pump.dm [hunk 23]
```diff
@@ -1,6 +1,6 @@
 /obj/item/weapon/gun/projectile/shotgun/pump
 	name = "FS SG \"Kammerer\""
-	desc = "Old Remington design from modern materials, this shotgun is a favourite of militia forces on many worlds."
+	desc = "When an old Remington design meets modern materials, this shotgun is being made. A favourite weapon of militia forces throughout many worlds."
```
Comment (NanakoAC) [inline code/modules/projectiles/guns/projectile/shotgun/pump.dm]: change to
"When an old Remington design meets modern materials, this is the result. A favourite weapon of militia forces throughout many worlds."

File: code/modules/projectiles/guns/projectile/sniper.dm [hunk 24]
```diff
@@ -1,6 +1,6 @@
 /obj/item/weapon/gun/projectile/heavysniper
 	name = "NT AMR \"Penetrator\""
-	desc = "A portable anti-armour rifle fitted with a scope, it was originally designed to used against armoured exosuits. It is capable of punching through windows and non-reinforced walls with ease. Fires armor piercing 14.5mm shells."
+	desc = "A portable anti-armour rifle, fitted with a scope, it was originally designed to use against armoured exosuits. It is capable of punching through windows and non-reinforced walls with ease. Fires armor piercing 14.5mm shells."
```
Comment (NanakoAC) [inline code/modules/projectiles/guns/projectile/sniper.dm]: change to "designed for use"

**Comment by Walfanger** [comment]
Well, got it. Thank you for your revising. Will do. 
This will improve my English for sure :^)

**Comment by NanakoAC** [comment]
Is this ready for review again?

File: code/game/objects/items/weapons/melee/energy.dm (hunk 1 above)
Comment (NanakoAC) [reply code/game/objects/items/weapons/melee/energy.dm]: this one is still incorrect, revert

File: code/modules/projectiles/guns/energy/laser.dm (hunk 4 above)
Comment (NanakoAC) [reply code/modules/projectiles/guns/energy/laser.dm]: Not quite right. should be "were made"

File: code/modules/projectiles/guns/energy/laser.dm [hunk 25]
```diff
@@ -36,7 +36,7 @@ obj/item/weapon/gun/energy/retro
 	name = "NT LG \"Destiny\""
 	icon_state = "caplaser"
 	item_state = "caplaser"
-	desc = "This weapon is old, yet still robust and reliable. Its marked with old Nanotrasen brand, like a last reminder of what this corporation was, before church take control of everything."
+	desc = "This weapon is old, yet still robust and reliable. It's marked with old Nanotrasen brand, a distant reminder of what this corporation was, before the Church take control of everything."
```
Comment (NanakoAC) [inline code/modules/projectiles/guns/energy/laser.dm]: should be "took control"

File: code/game/objects/items/weapons/grenades/explosive.dm [hunk 26]
```diff
@@ -47,7 +47,7 @@
 
 /obj/item/weapon/grenade/frag/explosive
 	name = "FS HEG \"Zoe\""
-	desc = "A military High Explosive grenade, designed wreak havoc in certan radius."
+	desc = "A military High Explosive grenade, designed to wreak havoc in a certan radius."
```
Comment (NanakoAC) [inline code/game/objects/items/weapons/grenades/explosive.dm]: "certain" not certan

File: code/game/objects/items/weapons/grenades/smokebomb.dm [hunk 27]
```diff
@@ -1,6 +1,6 @@
 /obj/item/weapon/grenade/smokebomb
 	name = "FS SG \"Reynolds\""
-	desc = "Smoke grenade, used to provide cover and hide movements."
+	desc = "Smoke grenade, used to create smoke cloud providing cover and hiding one's movement."
```
Comment (NanakoAC) [inline code/game/objects/items/weapons/grenades/smokebomb.dm]: try this instead

"Smoke grenade, used to create a cloud of smoke providing cover and hiding movement."

File: code/modules/projectiles/guns/energy/crossbow.dm [hunk 28]
```diff
@@ -1,6 +1,6 @@
 /obj/item/weapon/gun/energy/crossbow
 	name = "NT EC \"Nemesis\""
-	desc = "Mini energy crossbow, produced by old Nanotrasen, discontinued now. A weapon favored by many mercenary stealth specialists."
+	desc = "Mini energy crossbow, produced by the old Nanotrasen, discontinued now. A weapon favored by many mercenary stealth specialists."
```
Comment (NanakoAC) [inline code/modules/projectiles/guns/energy/crossbow.dm]: this was better before

File: code/modules/projectiles/guns/launcher/syringe_gun.dm [hunk 29]
```diff
@@ -63,7 +63,7 @@
 
 /obj/item/weapon/gun/launcher/syringe
 	name = "syringe gun"
-	desc = "A spring loaded rifle designed to fit syringes, designed to incapacitate unruly patients from a distance."
+	desc = "A spring-loaded rifle designed to shoot syringes to incapacitate unruly patients from a distance."
```
Comment (NanakoAC) [inline code/modules/projectiles/guns/launcher/syringe_gun.dm]: fire syringes

File: code/modules/projectiles/guns/projectile/automatic/ak47.dm [hunk 30]
```diff
@@ -1,8 +1,8 @@
 /obj/item/weapon/gun/projectile/automatic/ak47
 	name = "Excelsior 7.62x39 AKMS"
-	desc = "Weapon of oppressed, oppressors and just crazy terrorists.\
-		 If it doesn't work, you can always hit him with it! It is the really old designed assault rifle, chambered in 7,62x39.\
-		 It is known for it easy maintaining and low price. This gun is not used by military anymore, but it found a wide spread within criminals and insurgents."
+	desc = "Weapon of an oppressed, oppressors, and extremists of all flavours.\
+		 This is an ancient automatic rifle, chambered in 7.62x39. If it won't fire, percussive maintenance should get it working again.\
+		 It is known for its easy maintaining, and low price. This gun is not in active military service anymore, but has become ubiquitous among criminals and insurgents."
```
Comment (NanakoAC) [inline code/modules/projectiles/guns/projectile/automatic/ak47.dm]: this is still wrong, you only copied some of what i wrote before

should be "the oppressed"
"easy maintenance"

**Pull request merged**